src/managers/data_manager.py

```
@@ -747,51 +747,123 @@ def _build_flat_data_structure(self, raw_data: List[Dict]) -> bool:
+                logger.info(f"Flat structure built (fallback): {len(self.app_state['data_map'])} items")
+
             # 解析結果表示を更新
             if self.app_state.get("analysis_results"):
                 self._update_analysis_result_display(self.app_state["analysis_results"])
-            
+
             self._set_loading_state(False)
             return True
-            
+
         except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)